src/main/java/org/jboss/logmanager/MDC.java

```diff
@@ -16,16 +16,32 @@
 
 package org.jboss.logmanager;
 
+import java.util.Iterator;
 import java.util.Map;
+import java.util.ServiceConfigurationError;
+import java.util.ServiceLoader;
 
 /**
  * Mapped diagnostic context.  This is a thread-local map used to hold loggable information.
```

Comment:
I know, late to the party, but `This is a thread-local map` is now outdated or better to say only true by default.